### PR TITLE
Feature: #2 show variable names

### PR DESCRIPTION
**Pull request by uhbif19**
For #2 issue.

### CODE REVIEW

File: flake8_variables_names/checker.py [hunk 1]
```diff
@@ -67,13 +69,13 @@ def get_varname_errors(self, var_name: str, var_ast_node) -> List[Tuple[int, int
             errors.append((
                 var_ast_node.lineno,
                 var_ast_node.col_offset,
-                'VNE001 single letter variable names are not allowed',
+                'VNE001 single letter variable names like \'%s\' are not allowed' % (var_name,),
```
Comment (Melevir) [inline flake8_variables_names/checker.py]: Change quotes here to comply PEP8, please (https://www.python.org/dev/peps/pep-0008/#string-quotes)

File: flake8_variables_names/checker.py [hunk 2]
```diff
@@ -67,13 +69,13 @@ def get_varname_errors(self, var_name: str, var_ast_node) -> List[Tuple[int, int
...
             ))
         if var_name in self.variable_names_blacklist:
             errors.append((
                 var_ast_node.lineno,
                 var_ast_node.col_offset,
-                'VNE002 variable name should be clarified',
+                'VNE002 variable name \'%s\' should be clarified' % (var_name,),
```
Comment (Melevir) [inline flake8_variables_names/checker.py]: Same thing with quotes here.

Comment (Melevir) [reply flake8_variables_names/checker.py]: Also use `.format` to comply style guide (link can be found in readme).

**Review by Melevir — CHANGES_REQUESTED** [review]
Couple of small style fixes and we're all set.

File: flake8_variables_names/checker.py (hunk 2 above)
Comment (uhbif19) [reply flake8_variables_names/checker.py]: Fixed. Also fix docstrig.

It is not clear for me from your styleguide, if single string formatting param requires numbering too. I presumed that it does.

**Comment by Melevir** [comment]
Thanks a lot

**Pull request merged**